pages/instructorRequestCourse/instructorRequestCourse.sql

```diff
@@ -0,0 +1,48 @@
+-- BLOCK get_requests
+WITH
+select_course_requests AS (
+    SELECT
+        coalesce(jsonb_agg(
+            jsonb_build_object(
+                'status', r.approved_status,
+                'details',
+                    CASE
+                        WHEN r.approved_status = 'approved' THEN CONCAT('Approved by ', u.name)
+                        WHEN r.approved_status = 'denied'   THEN CONCAT('Denied by ', u.name)
+                        ELSE ''
+                    END,
+                'short_name', r.short_name,
+                'title', r.title)),
+        '[]'::jsonb) AS course_requests
```

Comment:
I assume you are only using JSON here because you also want to return `has_one_course`? If you get rid of the complex form preamble then you can replace this with a simple SQL query (no JSON).